src/ontology/cl-edit.owl

```diff
@@ -32517,6 +32518,19 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
 SubClassOf(obo:CL_4042027 obo:CL_0002614)
 SubClassOf(obo:CL_4042027 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001903))
 
+# Class: obo:CL_4042031 (immune oligodendroglia)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30747918") Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasDbXref "PMID:35662200") obo:IAO_0000115 obo:CL_4042031 "An oligodendrocyte of the central nervous system that exhibits immune properties such as non-self antigen presentation to T cells, phagocytosis of debris, and cytokine and chemokine production. This cell type is immunoreactive during inflammation, neurodegenerative disorders, chronic stress and major depressive disorder.")
```

Comment:
I would define it more a cell state than cell type. If you want to avoid the terminology, consider using only "Immune oligodendroglial is immunoreactive ..."